hs_app_netCDF/receivers.py

```diff
@@ -14,15 +14,17 @@
 from hs_core.hydroshare import utils
 from hs_app_netCDF.forms import VariableValidationForm, OriginalCoverageForm, VariableForm
 from hs_app_netCDF.models import NetcdfResource
-import nc_functions.nc_utils as nc_utils
-import nc_functions.nc_dump as nc_dump
-import nc_functions.nc_meta as nc_meta
+import hs_file_types.nc_functions.nc_utils as nc_utils
+import hs_file_types.nc_functions.nc_dump as nc_dump
+import hs_file_types.nc_functions.nc_meta as nc_meta
 
 
 # receiver used to extract metadata after user click on "create resource"
 @receiver(pre_create_resource, sender=NetcdfResource)
 def netcdf_pre_create_resource(sender, **kwargs):
-
+    # Pabitra: Though there is some common code between netcdf file type and here as it relates
```

Comment:
I will probably end up disagreeing with this in the future. I think there is a way we can make the way all metadata is stored consistent. This is more work than the composite resource but not impossible or impractical.